src/js/App/Header/Tools.js

```diff
@@ -19,12 +19,16 @@ import HeaderAlert from './HeaderAlert';
 import cookie from 'js-cookie';
 import './Tools.scss';
 import { isBeta } from '../../utils';
+import { spinUpStore } from '../../redux-config';
 import classnames from 'classnames';
 
 export const switchRelease = (isBeta, pathname) => {
   cookie.set('cs_toggledRelease', 'true');
+  const { store } = spinUpStore();
+  const isAppOnlyOnBeta = store.getState().chrome.activeSection.isBeta;
```

Comment:
need to use a optional chaining here:
 ```suggestion
  const isAppOnlyOnBeta = store.getState().chrome.activeSection?.isBeta;
```